apps/public-docsite-v9/.storybook/preview.js

```diff
@@ -0,0 +1,43 @@
+// import * as rootPreview from '../../../.storybook/preview';
+// import { FluentDocsContainer } from '../src/DocsComponents/FluentDocsContainer.stories';
```

Comment:
what about those comments ? can we remove that pls ?

Reply:
yup! i can